Makefile

```diff
@@ -1,14 +1,17 @@
 default: build
```

Comment:
The new convention defines a new default (building the docker image in addition to the binary).
If we are happy with that, we should remove this definition of default. 
Personally, I would remove the definition of default from the convention (if conventions defines different 'default', it will be a nightmare to know what is the actual default and in the case of osd-network-verifier, building docker each time when doing local dev is a loss of time)

Reply:
moved default target after boilerplate makefiles.